src/pages/subject/[subjectID].tsx

```diff
@@ -754,7 +748,7 @@ const SubjectDetails: NextPage<{
           setEditConnection={setEditConnection}
           setDeleteConnection={setDeleteConnection}
         />
-        <PeriodLogsSection
+        {/* <PeriodLogsSection
```

Comment:
Should this be a TODO or should it be remove all together

Reply:
It looks like they don’t want us to keep these features, but might as well just commented it out just in case they want them again.